src/lib/components/Cytoscape.react.js

```diff
@@ -423,6 +548,24 @@ class Cytoscape extends Component {
         document.body.removeChild(downloadLink);
     }
 
+    generateContextMenu(contextMenu) {
+        return this._cy.contextMenus({
```

Comment:
Doesn't need to be a `return` statement

Also I would call this function `updateContextMenu()`